e2e/cypress/utils/constants.js

```diff
@@ -13,6 +13,11 @@ export const CLOUD = 'Cloud';
 export const E20 = 'E20';
 export const TEAM = 'Team';
 
+export const FixedPublicLinks = {
```

Comment:
:thinking: What does "fixed" mean in this context? Static or it is a temporary fix? Anyway, this is non blocking. I think I just don't understand.

Reply:
It's hard to name things :) but in this context, the public links are hard-coded in our codebase. Other instances of terms of service and privacy policy links in the app are configurable via server config but not in "Edition and License" at system console page and some section of the app. Hence, I prefix it with "Fixed" but other name would work too. 0/5 on naming and very much open to more appropriate suggestion.